tests/e2e/nebulacluster_custom_config_test.go

```diff
@@ -0,0 +1,99 @@
+package e2e
+
+import (
+	"github.com/vesoft-inc/nebula-operator/tests/e2e/e2ematcher"
+	"sigs.k8s.io/e2e-framework/third_party/helm"
+
+	"github.com/vesoft-inc/nebula-operator/tests/e2e/envfuncsext"
+)
+
+const (
+	LabelCustomConfig        = "custom config"
+	LabelCustomConfigStatic  = "static"
+	LabelCustomConfigDynamic = "dynamic"
+)
+
+var testCasesCustomConfig []ncTestCase
+
+func init() {
+	testCasesCustomConfig = append(testCasesCustomConfig, testCasesCustomConfigStatic...)
+	testCasesCustomConfig = append(testCasesCustomConfig, testCasesCustomConfigDynamic...)
+}
+
+// test cases about static custom config
+var testCasesCustomConfigStatic = []ncTestCase{
+	{
+		Name: "custom config for static",
+		Labels: map[string]string{
+			LabelKeyCategory: LabelCustomConfig,
+			LabelKeyGroup:    LabelCustomConfigStatic,
+		},
+		InstallWaitNCOptions: []envfuncsext.NebulaClusterOption{
+			envfuncsext.WithNebulaClusterReadyFuncs(
+				envfuncsext.NebulaClusterReadyFuncForFields(false, map[string]any{
+					"Spec": map[string]any{
+						"Graphd": map[string]any{
+							"Replicas": e2ematcher.ValidatorEq(2),
+						},
+						"Metad": map[string]any{
+							"Replicas": e2ematcher.ValidatorEq(3),
+						},
+						"Storaged": map[string]any{
+							"Replicas": e2ematcher.ValidatorEq(3),
+						},
+					},
+				}),
+				envfuncsext.DefaultNebulaClusterReadyFunc,
+			),
+		},
+		LoadLDBC: true,
```

Comment:
don't need load ldbc?

Reply:
There will be a restart here. If there is data, it will be more realistic and it will be easier to find problems.